### DISCUSSION TITLE
Trouble installing 0.3.0.0 using the provided instructions

### DISCUSSION THREAD

**Discussion by tomtheisen** [net-lisias-ksp/KSP-Recall#51] [Support] [answered]
I'm a beginner at KSP mods, and KSP in general.  I'm having some trouble following the installation instructions in `INSTALL.md` for the 0.3.0.0 zip package.  Here's the excerpt I'm looking at.

> * **REMOVE ANY OLD VERSIONS OF THE PRODUCT BEFORE INSTALLING**, including any other fork:
>	+ Delete `<KSP_ROOT>/GameData/999_KSP-Recall`
> * Extract the package's `GameData/` folder into your KSP's as follows:
>	+ `<PACKAGE>/GameData/000_KSP-Recall/*` --> `<KSP_ROOT>/GameData/000_KSP-Recall`
>		- Overwrite any preexisting file.

And then it says this folder structure must be present.

>```
><KSP_ROOT>
>	[GameData]
>		[000_KSP-Recall]
>```

But in the zip file package, there's a `999_KSP-Recall` folder, not `000_KSP-Recall`.  It specifically says to delete `999` one.  Where do I get the `000` one?  Should I rename the folder from the zip?

**Comment by Lisias** [comment, marked as answer]
Humm… Sorry, the INSTALL instructions are wrong.

I will fix it for the next Release.

The correct foldername is `999_KSP-Recall`

**Reply by Lisias** [reply]
Thanks for the heads up, @tomtheisen .

I fixed the INSTALL instructions on commit https://github.com/net-lisias-ksp/KSP-Recall/commit/a9fdd64e30d08fc8515b5b42e77cd555e285aa79
